src/core/document.js

```diff
@@ -1005,6 +1006,21 @@ class PDFDocument {
       Promise.all(allPromises).then(() => allFields)
     );
   }
+
+  get calculationOrderIds() {
```

Comment:
This getter needs a unit test on `document_spec.js`. I think it can be very similar to https://github.com/mozilla/pdf.js/blob/master/test/unit/document_spec.js#L108-L118 where we provide an `AcroForm` dictionary with first no valid `CO` entry to test the `null` case and then with a valid `CO` entry to assert that the IDs are returned properly.

(Thinking about this, the `fieldObjects` getter should have gotten a similar unit test. If that can be added in a separate PR that would be great.)